code/style.css

```diff
@@ -1,4 +1,424 @@
 html {
-  background: #0025ff;
+  box-sizing: border-box;
+}
+
+/***Header for MOBILE***/
+
+header {
+  display: flex;
+  justify-content: center;
+  width: 100%;
+  border-bottom: 1px solid #000;
+}
+
+header nav {
+  display: none;
+}
+
+/*** Header for TABLET ***/
+
+@media (min-width: 668px) and (max-width: 1024) {
+  header {
+    display: flex;
+    flex-direction: row;
+    padding: 0 13px;
+    justify-content: flex-start;
+    align-items: center;
+  }
+
+  header nav {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    margin-left: auto;
+    width: 100%;
+    justify-content: center;
+  }
+
+  nav a {
+    font-family: "Lexend Giga", sans-serif;
+    text-decoration: none;
+    justify-content: space-between;
+    align-items: center;
+    margin-right: 20px;
+    color: black;
+    font-size: 15px;
+    font-weight: 50;
+  }
+
+  nav a:hover {
+    color: #f7941f;
+  }
+
+  .logo {
+    width: 70px;
+    height: 70px;
+    position: center;
+    flex-grow: 0;
+  }
+
+  .left {
+    order: 0;
+    display: flex;
+    align-items: flex-start;
+  }
+
+  .right {
+    order: 2;
+    display: flex;
+    align-items: flex-end;
+    margin-left: 20px;
+  }
+
+  .leftside {
+    display: inline-flex;
+  }
+
+  .rightside {
+    display: inline-flex;
+  }
+}
+/*** Header for DESKTOP ***/
+
+@media (min-width: 1024px) {
+  header {
+    background-color: #f7f7f7;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 0 13px;
+    position: fixed;
+    top: 0;
+    z-index: 4;
+  }
+  header nav {
+    display: flex;
+    margin-left: auto;
+    width: 50%;
+    justify-content: space-evenly;
+  }
+
+  nav a {
+    font-family: "Lexend Giga", sans-serif;
+    text-decoration: none;
+    color: black;
+    font-size: 15px;
+    font-weight: 50;
+  }
+
+  nav a:hover {
+    color: #f7941f;
+  }
+  .logo {
+    height: 70px;
+    width: 70px;
+  }
+}
+
+body {
+  font-family: arial, sans-serif;
+  margin: 0 auto;
+  background-color: white;
+}
+
+*,
+*:before,
+*:after {
+  box-sizing: inherit;
+}
```

Comment:
The body and *-styling should be at the top of the stylesheet, in this case at the top of the styling for @media (min-width: 1024px).